composer.json

```diff
@@ -22,16 +22,16 @@
         }
     ],
     "require": {
-        "php": "^5.6|^7.0",
+        "php": ">=7.0",
```

Comment:
This should be `^7.0`, as the current constraint would allow the installation on PHP 8 when it gets released which of course cannot be guaranteed. btw, do you plan to actually use PHP 7 stuff in the code (like return type-hints etc) now that PHP 7 is required? And another question, since neomerx v2 requires at least PHP 7.1 that means that v1 of this package will still use neomerx v1 - is neomerx v2 planned then for 2.0?

Reply:
Good spot - I'd copied the PHP constraint from Laravel 5.5 but you're totally right that it should be `^7.0`.

For `1.0` I'm planning on putting the minimum PHP version as `7.1` though haven't done it on this branch because it's not technically required for this set of changes. Will change that on `develop` prior to release.

I'd prefer to get `1.0` tagged without too many breaking changes (which is why a lot of stuff is now being deprecated rather than removed). So the plan is not to use PHP 7 language features for existing code, but anything new can use it. For the same reason, `1.0` will continue to use `neomerx/json-api:^1.0`.

For `2.0` I'll clean out all the deprecated code (there's a lot of it now), change everything to use PHP 7 language features, plus upgrade to `neomerx/json-api:^2.0`.